content/en/docs/Advanced/customizing.md

```diff
@@ -131,6 +131,18 @@ install:
 ...
 ```
 
+To do this after installation, simply add a cloud config file in the `/oem` folder, for instance, to make `/var/lib/docker` persistent:
+
+```yaml
+#cloud-config
+
+stages:
+  rootfs:
+  - name: "user_custom_mount"
+    environment_file: "/run/cos/custom-layout.env"
```

Comment:
Is this file created with the contents of the `environment` key below, or is this file read in addition to what is defined in the key below?

Reply:
both. if not existing it will create a new file, but if does exist it will add that variable to it